docs/HOWTO/90_Misc/Unpack_dvd.rst

```diff
@@ -38,11 +53,47 @@ It then creates a link to the major version.  If version 7.6 is extracted,
 
       yum clean all && yum makecache
 
-Use the help option :code:`unpack_dvd --help` to see all options available for :program:`unpack_dvd`.
-
 .. WARNING::
 
-   At this time :program:`unpack_dvd` does not work with EL8 ISOs.  EL8 introduced
-   modules to repositories and :program:`unpack_dvd` can not handle these.
-   It will extract some but not all of RPMs on an EL8 ISO.
+   At this time :program:`unpack_dvd` does not work entirely with EL8 ISOs.
+   EL8 introduced modules to repositories and :program:`unpack_dvd` can not handle these.
+   It will extract some but not all of RPMs on an EL8 ISO. The files extracted
+   are enough to kickstart a basic EL8 system but some of the application stream
+   files are not extracted and added to the repository correctly.
+
+Extract PXE files
+-----------------
+
+Extracting the PXE files along with the OS files, was added to :program:`unpack_dvd` in :package:`simp-utils-6.4.0`.
+
+By default :program:`unpack_dvd` will pull information off the ISO and, using this information, create a directory named <os-family>-<version>-<arch>  under the tftpboot rsync directory and extract the PXE files there.
+
+The default rsync directory is :file:`/var/simp/environments/production/rsync/<os family>/Global/tftpboot/linux-install/`.  Options exist to change the environment in the rsync directory or to specify an alternate directory.
+
+The rsync directory or the directory you specified must exist before running :program:`unpack_dvd`.
+
+The following example will
+
+* extract the RPMs as described Extract OS Packages above to :file:`/var/www/yum/CentOS/8.0.1905`
+* extract the PXE files to :file:`/var/simp/environments/test/rsync/CentOS/Global/tftpboot/linux-install/centos-8.0.1905-x86_64`
+
+.. code:: bash
+
+   # Place the -X options after the ISO name
+   unpack_dvd -v 8.0.1905 /myisodir/CentOS--x86_64-1905-dvd1.iso -X -e test
+
+The following example will
+
+* extract the RPMs as described Extract OS Packages above to :file:`/my/repodir/yum/CentOS/8.0.1905`
+* extract the PXE files to :file:`/my/tftpboot/`
+
+.. code:: bash
+
+   # The PXE directory must follow the -X option.
+   unpack_dvd -v 8.0.1905 -d /my/repodir /myisodir/CentOS--x86_64-1905-dvd1.iso -X /my/tftpboot
+
+.. NOTE::
+
+   You cannot just extract the PXE files.  It will always extract the packages
```

Comment:
There is a `--no-unpack-yum` option to exclude package extraction.  Does this not work?